vesc_driver/include/vesc_driver/vesc_driver.hpp

```diff
@@ -68,14 +67,14 @@ class VescDriver
     CommandLimit(
       rclcpp::Node * node_ptr,
       const std::string & str,
-      const boost::optional<double> & min_lower = boost::optional<double>(),
-      const boost::optional<double> & max_upper = boost::optional<double>());
+      const std::optional<double> & min_lower = std::optional<double>(),
+      const std::optional<double> & max_upper = std::optional<double>());
```

Comment:
If we are going to use `std::optional` then this is missing `#include <optional>`.